qiskit/visualization/counts_visualization.py

```diff
@@ -198,7 +198,7 @@ def plot_histogram(
     # add some text for labels, title, and axes ticks
     ax.set_ylabel("Probabilities", fontsize=14)
     all_vals = np.concatenate(all_pvalues).ravel()
-    ax.set_ylim([0, min(1.2, max(1.2 * val for val in all_vals))])
+    ax.set_ylim([0.0, min([1.2, max([1.2 * val for val in all_vals])])])
```

Comment:
I'm honestly surprised that pylint didn't tell you to use a generator here.

Reply:
I was was told in the original PR that also set the lower bound to neg.  But once I reverted that back to the original code it did not care.  Better than Black wanting to format twice because of a header change though....